virtio-devices/src/seccomp_filters.rs

```diff
@@ -159,6 +159,7 @@ fn virtio_pmem_thread_rules() -> Result<Vec<SyscallRuleSet>, Error> {
 
 fn virtio_rng_thread_rules() -> Result<Vec<SyscallRuleSet>, Error> {
     Ok(vec![
+        allow_syscall(libc::SYS_brk),
```

Comment:
Cool! Can you actually add `brk()` to every seccomp filter? I think that's something that will come bite us later otherwise.

Reply:
That's a good point. I just added a new commit to address this issue. :)